src/Lights/Shadows/shadowGenerator.ts

```diff
@@ -797,7 +797,7 @@ export class ShadowGenerator implements IShadowGenerator {
     protected _cachedPosition: Vector3 = new Vector3(Number.MAX_VALUE, Number.MAX_VALUE, Number.MAX_VALUE);
     protected _cachedDirection: Vector3 = new Vector3(Number.MAX_VALUE, Number.MAX_VALUE, Number.MAX_VALUE);
     protected _cachedDefines: string;
-    protected _currentRenderID: number;
+    protected _currentRenderId: number;

```

Comment:
protected variable => no backward-compatibilty issue except if this class is intended to be derived from by the user, which I have no idea. Your take @RaananW  ?

Reply:
fine with me.